supabase/migrations/20260901140000_updates_cache_invalidation.sql

```
@@ -0,0 +1,228 @@
+-- per-app token bump out to every regional plugin worker. The cache TTL
+-- remains the backstop if a call is lost.
+--
+-- All triggers are STATEMENT-level with transition tables: bulk writes
+-- (daily channel_devices cleanup, bundle uploads inserting hundreds of
+-- manifest rows, org-wide backfills) produce ONE aggregated, deduplicated
+-- notification per statement instead of one per row.
+
+-- Notify helper: dedupes, caps and chunks the app list so the payload is
+-- always bounded (the fan-out endpoint enforces 100 ids per request; an
+-- org bigger than the cap falls back to the cache TTL for the tail).
+CREATE OR REPLACE FUNCTION public.notify_updates_cache_invalidation(p_app_ids text[])
```

SQLFluff (4.2.2)

[error] Line is too long (85 > 80).

(LT05)